src/cascadia/TerminalApp/MinMaxCloseControl.cpp

```diff
@@ -16,59 +17,37 @@ namespace winrt::TerminalApp::implementation
         InitializeComponent();
     }
 
-    uint64_t MinMaxCloseControl::ParentWindowHandle() const
+    void MinMaxCloseControl::Maximize()
     {
-        return reinterpret_cast<uint64_t>(_window);
+        VisualStateManager::GoToState(MaximizeButton(), L"WindowStateMaximized", false);

```

Comment:
Is this (`L"WindowStateMaximized"`) some magic string that doesn't have a constant we could use instead?

Reply:
I'm not totally sure. It's defined in XAML so I don't think we can make it a constant, unless @DHowett-MSFT knows better than me.

Reply:
Alas, best we can do is "known shared value between XAML and code" ... even if we make it a constant, we'd need to keep it in sync. Oh, we made this one. Xaml didn't.

Reply:
Sorry: it is in the xaml file that we authored, not in "framework xaml", so _we_ own it and need to keep it in sync

Reply:
Okay, good to know. Thanks Dustin!